### PR TITLE
docs: remove tsh recording exports

### PR DESCRIPTION
**Pull request by stevenGravy**
This is no longer applicable for v15+.

### CODE REVIEW

File: docs/pages/desktop-access/reference/sessions.mdx [hunk 1]
```diff
@@ -64,8 +64,18 @@ desktop icon in the first column to distinguish them from SSH recordings.
 
 ![Desktop Session Recording](../../../img/desktop-access/session-recording@2x.png)
 
-Click the play button to open the player in a new tab. To export desktop session
-recordings to video for playback outside of Teleport, use the
+Click the play button to open the player in a new tab.
```
Comment (zmb3) [inline docs/pages/desktop-access/reference/sessions.mdx:67]: Let's just remove the whole snippet about `tsh recordings export`.

Comment (stevenGravy) [reply docs/pages/desktop-access/reference/sessions.mdx:67]: Removed @zmb3 , take out this too? https://goteleport.com/docs/reference/cli/tsh/#tsh-recordings-export

Comment (zmb3) [reply docs/pages/desktop-access/reference/sessions.mdx:67]: Sure

Comment (stevenGravy) [reply docs/pages/desktop-access/reference/sessions.mdx:67]: The reference to tsh recordings export has been remove, please approve after reviewing.

**Review by probakowski — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]

**Pull request merged**